factcast-store-pgsql/src/test/resources/test_schema.sql

```diff
@@ -10,6 +10,9 @@ DROP INDEX IF EXISTS idx_fact_unique_id;
 DROP TABLE IF EXISTS fact CASCADE;
 DROP TABLE IF EXISTS catchup CASCADE;
 
+DROP TABLE IF EXISTS schemastore cascade;
```

Comment:
not a problem here, but in general not a good pattern. couldn't we just add indexes and constraints if they do not exist?

Reply:
Mh. I guess we have to make sure that the tables are cleaned/truncated because otherwise you dont have a "clean" state between multiple test executions 🤔A `DbCleaner` you know 🙈

Reply:
grrrrrr... :laughing: